lib/ansible/modules/network/f5/bigip_command.py

```diff
@@ -78,6 +78,17 @@
         conditional, the interval indicates how to long to wait before
         trying the command again.
     default: 1
+  transport:
```

Comment:
Is it better to add `transport` as part of common parameters?

Reply:
That's a good suggestion, though only if `transport` is truly supported by all F5 modules

Reply:
I was specifically told to move it to the module here to avoid breaking other F5 modules.

Reply:
yes, do **not** add it to all modules.